lib/iiif_manifest/v3/manifest_builder.rb

```diff
@@ -29,8 +29,32 @@ def to_h
         @to_h ||= builders.new(work).apply(top_record)
       end
 
+      # Utility method to wrap the obj into a IIIF V3 compliant language map as needed.
+      def self.language_map(obj)
+        # self.valid_language_map?(obj) ? obj : self.transform(obj)
+        obj.is_a?(Hash) ? transform_hash_value(obj) : transform_obj_value(obj)
+      end
+
+      private_class_method
+        def self.transform_obj_value(obj)
```

Comment:
I know that I named these originally, but it might be good to give them a more specific name that gives a bit more context.  Maybe `transform_language_obj_value`?